authentication/account/repository/identity.go

```diff
@@ -135,7 +135,7 @@ type IdentityRepository interface {
 	Query(funcs ...func(*gorm.DB) *gorm.DB) ([]Identity, error)
 	List(ctx context.Context) ([]Identity, error)
 	ListIdentitiesToNotifyForDeactivation(ctx context.Context, lastActivity time.Time, limit int) ([]Identity, error)
-	ListIdentitiesToDeactivate(ctx context.Context, lastActivity, notification time.Time, limit int) ([]Identity, error)
+	ListIdentitiesToDeactivate(ctx context.Context, lastActivity, notification time.Time, exclusions []string, limit int) ([]Identity, error)
```

Comment:
Let's use either `whitelist` or `exclusions` everywhere :) Just pick one.
BTW, `exclusions` is pretty clear in this context. But it's your call. Both are fine.

Reply:
you're right, that's a left-over that I will revert to `whitelist` to be consistent with the rest of the changes

Reply:
done in https://github.com/fabric8-services/fabric8-auth/pull/859/commits/c7ab003a977dc205ea5c62b7a26b319ede8840dd